src/widgets/principles/ui/principle-card/principles.scss

```
@@ -13,19 +13,16 @@
 
   & .cards {
```

please remove all parent selectors

```suggestion
.cards {
```